<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EIPPreferences</class>
 <widget class="QDialog" name="EIPPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encrypted Internet Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../../data/resources/icons.qrc">
    <normaloff>:/images/mask-icon.png</normaloff>:/images/mask-icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbGatewaySelector">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Select gateway for provider</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblSelectProvider">
        <property name="text">
         <string>Select &amp;provider:</string>
        </property>
        <property name="buddy">
         <cstring>cbProvidersGateway</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cbProvidersGateway">
        <item>
         <property name="text">
          <string>&lt;Select provider&gt;</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="pbSaveGateway">
        <property name="text">
         <string>&amp;Save this provider settings</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="lblProvidersGatewayStatus">
        <property name="text">
         <string>&lt; Providers Gateway Status &gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select &amp;gateway:</string>
        </property>
        <property name="buddy">
         <cstring>cbGateways</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="cbGateways">
        <item>
         <property name="text">
          <string>Automatic</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbProvidersGateway</tabstop>
  <tabstop>cbGateways</tabstop>
  <tabstop>pbSaveGateway</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../data/resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
